doc/releases/changelog-dev.md

```diff
@@ -167,6 +166,9 @@ array([False, False])
 * `transmon_drive` is updated in accordance with [1904.06560](https://arxiv.org/abs/1904.06560). In particular, the functional form has been changed from $\Omega(t)(\cos(\omega_d t + \phi) X - \sin(\omega_d t + \phi) Y)$ to $\Omega(t) \sin(\omega_d t + \phi) Y$.
   [(#4418)](https://github.com/PennyLaneAI/pennylane/pull/4418/)
 
+* `StatePrep` is renamed to `StatePrepBase` and `QubitStateVector` is renamed to `StatePrep`. 
+  [(#4450)](https://github.com/PennyLaneAI/pennylane/pull/4450)
+
```

Comment:
Shouldn't this fall under breaking changes?

Reply:
Since `QubitStateVector` is still accessible and `StatePrep` can be accessed using `qml.operation.StatePrep`, it doesn't really break anything? 

Although a user might be using logic that depends on `op.name` or `op.__str__` that might break

Reply:
Given plugins tend to depend on the name, might be better safe than sorry and leaving a comment under "Breaking changes".

Reply:
iirc there's a plan to deprecate that name in the future. I'd say we can add it to the deprecations page (and maybe the breaking changes section) whenever that happens. Although, for this release, we should update this changelog note to mention that `QubitStateVector` is still accessible (for now?) to avoid panic